### ISSUE TITLE
Suggestion: use fzf's --tac option instead of tac binary

### ISSUE THREAD

**Issue by mspielberg** [bigH/interactively#1]
This eliminates a dependency on coreutils for systems that do not have it installed by default, e.g. BSD variants including macOS.

**Comment by bigH** [comment]
I agree. I don't have the time to fix this right now. Feel free to make a PR.
